raiden_contracts/contracts/MonitoringService.sol

```diff
@@ -0,0 +1,321 @@
+pragma solidity ^0.4.23;
+
+import "./Token.sol";
+import "./Utils.sol";
+import "./lib/ECVerify.sol";
+import "./TokenNetwork.sol";
+
+contract MonitoringService is Utils {
+
+    // Token to be used for paying the rewards
+    Token public token; // Only allow RDN?
+
+    // The minimum allowed deposit for a MS
+    uint256 public minimum_deposit;
+
+    // keep track of registered Monitoring Services
+    // A Monitoring Services must deposit to register
+    mapping(address => bool) public registered_monitoring_services;
```

Comment:
The current design should allow for MS addresses to also be an MS user. This is allowed as long as they have more tokens deposited than the `minimum_deposit`. But the point about separating the two kinds of deposits is valid if we want to use the deposit for the entire Service Bundle.

Reply:
Ah, you are right about the current design.

> But the point about separating the two kinds of deposits is valid if we want to use the deposit for the entire Service Bundle.

I am assuming this because we do not have specs for any other contract at this point and the Raiden Bundle needs to deposit anyway + the MS needs to deposit for it to be considered registered.
This being said, I'm also pinging @LefterisJP here, because it seems to be an issue that's not clearly specced out.

Reply:
Yes this is something we need to discuss and spec further. The RSB idea requires a single deposit per operator of a Bundle. So there should be a different contract holding the deposits. How would that looks remains TBD. Made this issue to track the discussion: https://github.com/raiden-network/spec/issues/79